### DISCUSSION TITLE
MQTT wo "Gesamtertrag Insgesamt", "Gesamtertrag Heute", "Gesamtleistung" ?

### DISCUSSION THREAD

**Discussion by HaraldNoack** [tbnobody/OpenDTU#537] [Q&A]
Hallo,

ich kann mit "MQTT Explorer" folgende Topics nicht finden:
"Gesamtertrag Insgesamt"
"Gesamtertrag Heute"
"Gesamtleistung"

Werden diese nicht übertrage ?
Ich will diese auf einem MQTT Dashboard darstellen.
Würde diese "Summenwerte" unter "solar/dtu" oder "solar/all" suchen.

Danke!!!!!

**Comment by Juergen2453** [comment]
Gesamtertrag Insgesamt = solar/{seriennummer-inverter}/0/yieldtotal
Gesamtertrag Heute        = solar/{seriennummer-inverter}/0/yieldday
Gesamtleistung                = solar/{seriennummer-inverter}/0/power

**Reply by HaraldNoack** [reply]
Danke für die Antwort, aber diese Felder sind Wechselrichter-spezifisch und nicht die Summenwerte über alle Wechselrichter.

Ich habe inzwischen herausgefunden, dass die Summenwerte zwar auf der Web-GUI angezeigt, aber nicht via MQTT übertragen werden.

**Reply by Juergen2453** [reply]
Ach so, mehrere Inverter. Per API Aufruf bekommst du die Werte.
z.B. curl http://192.168.10.122/api/livedata/status | jq

Die Dokumentation dazu

https://github.com/tbnobody/OpenDTU/tree/master/docs
https://github.com/tbnobody/OpenDTU/blob/master/docs/Web-API.md

**Reply by Hans-JR** [reply]
Auch ich habe nach diesen Daten gesucht, da ich demnächst 2 Inverter haben werde. Es währe schön, wenn diese Daten auch per MQTT  übertragen würden. Das würde die Auswertung natürlich sehr vereinfachen. Ich bin z. B. nicht in der Lage so problemlos den Datenstring zu zerlegen. Evtl. werde ich das mal per Blockly (ioBroker) versuchen.

**Reply by Juergen2453** [reply]
Mach einfach ein issues auf und warte auf eine Antwort.

**Reply by tbnobody** [reply]
Oder du wartest auf #350 oder #317 :)

**Reply by HaraldNoack** [reply]
Danke, gibt es für #350 einen Zeithorizont ?
